src/Altinn.App.Api/Controllers/OptionsController.cs

```diff
@@ -55,5 +59,42 @@ public OptionsController(IAltinnApp altinnApp, IAppOptionsService appOptionsServ
 
             return Ok(appOptions.Options);
         }
+
+        /// <summary>
+        /// Exposes options related to the app and logged in user
```

Comment:
I guess the option is closer related to the instance that the logged in user as we pass the instanceIdentifier?